docs/tutorials/chase-the-basketball.md

```diff
@@ -0,0 +1,1517 @@
+    ddddd111dddddddddddddddddddddddddddddddddddddddd1111ddddddddddddddddd11111111111111111111111ddddddddddddddddd1111ddddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddddd1111ddddddddddddddddd11111111111111111111111ddddddddddddddddd1111ddddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddddd111dddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddd111ddddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111ddddddddddddddddddddddddddddddddddddddd1111dddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddd1111dddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111ddddddddddddddddddddddddddddddddddddddd111ddddddddddddddddddd111ddddddddddddddddd111ddddddddddddddddddd111dddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111ddddddddddddddddddddddddddddddddddddddd111ddddddddddddddddddd1111ddddddddddddddd1111ddddddddddddddddddd111dddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd1111ddddddddddddddddddd1111ddddddddddddddd1111ddddddddddddddddddd1111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd1111ddddddddddddddd1111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd1111d1ddddddddddd1d1111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111dd1d1d1d1d1d1d1dd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111dd1d1d1d1d1d1d1dd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111dddd1d1d1d1d1dddd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    ddddd111dddddddddddddddddddddddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddd111dddddddddddddddddddd111ddddddddddddddddddddddddddddddddddddd111ddddd
+    `)
+```
+
+## Step 3
+
+Open the `||sprites:Sprites||` Toolbox drawer and drag the first block, `||variables:set mySprite||` into the `||loops:on start|` block on your Workspace. This will create a new `||sprites:Player||` character for your game.
```

Comment:
```suggestion
Open the `||sprites:Sprites||` Toolbox drawer and drag the first block you see, `||variables:set mySprite||`, into the `||loops:on start|` block on your Workspace. This will create a new `||sprites:Player||` character for your game.
```